test/e2e/serial/tests/resource_accounting.go

```diff
@@ -645,7 +647,8 @@ var _ = Describe("[serial][disruptive][scheduler][resacct] numaresources workloa
 			rl := e2ereslist.FromGuaranteedPod(*updatedPod2)
 			klog.Infof("post-create pod resource list: spec=[%s] updated=[%s]", e2ereslist.ToString(e2ereslist.FromContainers(podGuanranteed.Spec.Containers)), e2ereslist.ToString(rl))
 
-			policyFuncs := tmPolicyFuncsHandler[nrtv1alpha2.TopologyManagerPolicy(targetNrtInitial.TopologyPolicies[0])]
+			targetNrtInitialTopologyPolicies := noderesourcetopology.GetTopologyPolicies(*targetNrtInitial)
+			policyFuncs := tmPolicyFuncsHandler[nrtv1alpha2.TopologyManagerPolicy(targetNrtInitialTopologyPolicies[0])]
```

Comment:
I think `tmPolicyFuncsHandler` is keyed by the old `topologyPolicy` value, which doesn't (nor it should) directly translate to the policy/scope attribute pair (applies also elsewhere)